lib/widgets/atoms/profile_header.dart

```diff
@@ -18,52 +16,44 @@ class ProfileHeader extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final ThemeData theme = Theme.of(context);
+    var backgroundImage = avatarUrl != null
+        ? NetworkImage(avatarUrl!) as ImageProvider<Object>
+        : const AssetImage(Assets.blankAvatar);
+
     return Padding(
-      padding: const EdgeInsets.all(Insets.paddingExtraLarge),
+      padding: const EdgeInsets.fromLTRB(Insets.paddingExtraLarge,
+          Insets.paddingExtraLarge, Insets.paddingExtraLarge, 0),
       child: Row(
         crossAxisAlignment: CrossAxisAlignment.center,
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
-          Flexible(
-            child: Padding(
-              padding: const EdgeInsets.all(Insets.paddingExtraSmall),
-              child: Text(
-                profileMessage,
-                textAlign: TextAlign.start,
-                softWrap: true,
-                style: theme.textTheme.titleLarge?.copyWith(
-                  color: theme.colorScheme.onSurface,
-                ),
-              ),
+          ClipRRect(
+            borderRadius: BorderRadius.circular(Radii.roundedRectRadiusSmall),
+            child: Image(
+              image: backgroundImage,
+              width: Insets.paddingExtraLarge,
+              height: Insets.paddingExtraLarge,
+              fit: BoxFit.cover,
             ),
           ),
-          Stack(
-            children: [
-              if (profileCompletionPercentage != null)
-                SizedBox(
-                  width: Radii.avatarRadiusSmall * 2,
-                  height: Radii.avatarRadiusSmall * 2,
-                  child: CircularProgressIndicator(
-                    value: profileCompletionPercentage! / 100,
-                  ),
-                ),
-              InkWell(
-                onTap: () {
-                  //TODO(m-rosario): Remove this once the real logout flow is defined by UX
-                  Provider.of<ScaffoldModel>(
-                    context,
-                    listen: false,
-                  ).setTestHomeScaffold();
-                  Scaffold.of(context).openDrawer();
```

Comment:
It would be great if you included this in the revision so that we have a way to logout during testing.

Reply:
Sounds good!